README.md

```diff
@@ -7,4 +7,12 @@ Core repository for the platform team 4 is developing
 1. Run `yarn` to fetch dependencies.
 2. Run `yarn run start:dev` to simultaneously run the frontend and backend.
 
+# Running with Docker
+
+Running with Docker will ensure that users can quickly get environment up and running so they can quickly iterate on work. The application will work regardless of operating system and configuration.
+
+Ensure you have set up backend env files first. Checkout backend README for more info.  
+
+1. Run `docker-compose up` to simultaneously run the frontend and backend in docker containers.  
```

Comment:
forget if `docker-compose up` autopulls images that it hasn't downloaded. maybe `docker-compose pull && docker-compose up` if this isn't the case

Reply:
it does afaik